build_tools/cmake/iree_lit_test.cmake

```diff
@@ -59,7 +59,8 @@ function(iree_lit_test)
   endforeach(_DATA_DEP)
 
   # We run all our tests through a custom test runner to allow setup and teardown.
```

Comment:
This comment is drifting further from the thing it's explaining. Can you shift it down? Actually can it go directly above run_test? CMake doesn't get upset if there's a comment in the function call, does it?

Reply:
Didn't build locally, but the cmake configuration does not struggle with it. Let's check with the CI :) Further moved a similar comment in `iree_cc_test`.